frictionless/schema.py

```diff
@@ -270,6 +270,20 @@ def from_jsonschema(profile):
 
         return schema
 
+    def to_excel_template(self, xlsx_path: str) -> any:
+        """Export schema data to excel
```

Comment:
I would not use word `data` here as a schema is a metadata. Just `export schema as an excel template` I think